db/r2ghidra-sleigh

```diff
@@ -23,6 +23,8 @@ R2PM_INSTALL() {
 		rm -rf r2ghidra_sleigh
 		mv r2ghidra_sleigh-${R2G_VERSION} r2ghidra_sleigh
 		if [ "${GLOBAL}" = 1 ]; then
+			echo "Moving r2ghidra_sleigh in $(r2 -HR2_LIBR_PLUGINS)"
```

Comment:
Make this conditional above and just mkdir the system path instead of doing all the work on someone's user and then moving it to system